src/components/Accordion/index.js

```diff
@@ -20,6 +21,7 @@ const Accordion = React.forwardRef(({
       ref={ref}
       id={id}
       activeIndex={activeIndex}
+      dataTestId={dataTestId}
```

Comment:
Looks like we should pass this attribute as `data-testid={dataTestId}` to Prime components. Otherwise incorrect attribute `datatestid` will be applied to html tags instead of `data-testid`

Reply:
For `Datepicker` component you should assign `data-testid: dataTestId` in `getDefaultProps` function. So we pass `dataTestId` prop to Sphere UI component and then pass `data-testid` prop to Prime component.
> So, why datatestid is incorrect?

Technically `datatestid` attribute is not incorrect, but it's not a html `data-*` attribute. Also `data-testid` attrbute is used by different testing frameworks and kind of standard for selectors in tests.

Reply:
Key with dash is wrong for object. But correct `"data-testid": dataTestId`. Is it ok?

Reply:
Only for `Datepicker `

Reply:
Are you sure these 'data-*' properties are needed here? Why can't classes or component id's be used for tests? Well or you first need to reach a general agreement and change it everywhere after. IMHO ))

Reply:
> But correct "data-testid": dataTestId. Is it ok?

Yes, you should use string literals explicitly here

>Are you sure these 'data-*' properties are needed here? Why can't classes or id's be used for tests?

Using special attributes for selectors in tests is considered as a best practice, cypress for example: https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
What do you mean? I added that prop for each one component in sphere-ui.

Reply:
Not all components allow the transfer data-* attributes. Have you checked what really appears in DOM for everyone?

Reply:
I see contradiction in your comment. If there are components that not allow transfer data-* attributes and how can I test anyone of it?